recipes/multifunctional/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "multifunctional" %}
+{% set version = "0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/multifunctional-{{ version }}.tar.gz
+  sha256: 0dd9df32f0aec730d8e61fa8c81405cdd9cd568a86908a1d857a89d7de1733a3
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - setuptools >=68.0
+    - pip
+  run:
+    - python >=3.9
+    - bw2data >=4.0.dev45
+    - bw2io >=0.9.dev28
+    - bw_processing >=0.9.5
```

Comment:
[`bw_processing` is 1.0](https://github.com/conda-forge/bw_processing-feedstock/pull/12)